libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -118,6 +118,7 @@ export type CryptoCurrency = CurrencyCommon & {
     link: string;
   };
   deviceTicker?: string;
+  explorerId?: string;
```

Comment:
Please add a simple comment to explain a bit what this is for.

Reply:
Done